### PR TITLE
Add integration tests for `post_update_data`

### PR DESCRIPTION
**Pull request by dotfloat**
Resolves: #1671

### CODE REVIEW

**Comment by jondequinor** [comment]
Still WIP? Maybe set the status to Draft.

**Comment by dotfloat** [comment]
> Still WIP? Maybe set the status to Draft.

Outdated PR body that I forgot to delete when un-drafting the PR.

File: tests/storage/test_extraction.py [hunk 1]
```diff
@@ -288,6 +290,66 @@ def test_observations(client):
     assert obs["transformation"] is None
 
 
+def test_observation_transformation(client):
+    data = pd.DataFrame([[1, 0.1], [2, 0.2], [3, 0.3], [4, 0.4]], index=[0, 1, 2, 3])
+
+    print(os.getcwd())
```
Comment (oysteoh) [inline tests/storage/test_extraction.py:296]: debug purposes or is it needed?

**Comment by oysteoh** [comment]
Other than my small comment regarding the print-statment this looks perfectly fine to me 🚀

File: tests/storage/test_extraction.py (hunk 1 above)
Comment (dotfloat) [reply tests/storage/test_extraction.py]: Not needed. Removed.

**Review by oysteoh — APPROVED** [review]
Looks good to me ! 🚀

**Pull request merged**